components/public-api-server/pkg/apiv1/team_test.go

```diff
@@ -140,6 +140,44 @@ func TestTeamsService_ListTeams(t *testing.T) {
 	})
 }
 
+func TestTeamService_GetTeam(t *testing.T) {
+	var (
+		teamID = uuid.New().String()
+	)
+
+	t.Run("returns invalid argument when empty ID provided", func(t *testing.T) {
+		_, client := setupTeamService(t)
+
+		_, err := client.GetTeam(context.Background(), connect.NewRequest(&v1.GetTeamRequest{
+			TeamId: "",
+		}))
+		require.Error(t, err)
+		require.Equal(t, connect.CodeInvalidArgument, connect.CodeOf(err))
+	})
```

Comment:
Why have `t.Run`s nested inside the test rather than separate tests?

Reply:
Mostly personal preference such that the tests for a single RPC are all grouped under a single block. No other reason really.